sdks/python/pyproject.toml

```diff
@@ -26,7 +26,7 @@ requires = [
     # Avoid https://github.com/pypa/virtualenv/issues/2006
     "distlib==0.3.7",
     # Numpy headers
-    "numpy>=1.14.3,<2.2.0", # Update setup.py as well.
+    "numpy>=1.14.3,<1.27", # Update setup.py as well.
```

Comment:
Please revert this change (I'm assuming this was unintentional)

Reply:
Huh, yeah I didn't change this intentionally. Maybe weird git operation artifact. Reverting